compoments/report-repair/summary.js

```diff
@@ -23,6 +23,12 @@ const Summary = ({values, getNextStepFromCondition, submit, goToStep}) => {
     { pageName:'What best describes the problem?', value: values['repairProblemBestDescription']?.display, link: repairProblemBestDescriptionLink},
     { pageName:'Description', value: values.description?.text, link:'repair-description'}
   ]
+  if (process.env.releaseVersion == 'full'){
```

Comment:
Would it make sense to put `'full'` and `mvp` in a global variable?

Reply:
...and/or have a function which determines if a version is set, i.e. `isFullReleaseVersion` and `isMvpReleaseVersion`.
This would result in only a single instance of each value and make it easier to identify usages (for potential removal later).

Reply:
I've added helper functions for this in helpers/features